web/html/src/manager/state/recurring-states-list.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable */
+'use strict';
+
+const React = require("react");
+const ReactDOM = require("react-dom");
+const { InnerPanel } = require('components/panels/InnerPanel');
+const Button = require("components/buttons").Button;
+const {Toggler} = require("components/toggler");
+const ModalButton = require("components/dialog/ModalButton").ModalButton;
+const DeleteDialog = require("components/dialog/DeleteDialog").DeleteDialog;
+
+class RecurringStatesList extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            itemsToDelete: []
+        };
+    }
+
+    selectToDelete(item) {
+        this.setState({
+            itemToDelete: item
+        });
+    }
+
+    tableBody = () => {
+        const elements = [];
+        const data = this.props.data;
+        if (data !== undefined) {
+            data.map(row => {
+                elements.push(
+                    <tr>
+                        <td>
+                            <Toggler value={row.active} className="btn" handler={() => {this.props.onToggleActive(row)}} />
+                        </td>
+                        <td className="text-center">{row.scheduleName}</td>
+                        <td className="text-center">{row.cron}</td>
+                        {/*<td className="text-center">{row.createdAt + " " + timezone}</td>*/}
+                        {this.props.disableCreate ? <td className="text-center">{row.targetType}</td> : null}
+                        {/*<td className="text-center">
+                         <input type="checkbox" className="checkbox btn-group" checked={row.skipNext === "true"} onChange={() => {this.props.onSkip(row)}}/>
+                         </td> } */ }
+                        <td className="text-right">
+                            <div className="btn-group">
+                                <Button
+                                    className="btn-default btn-sm"
+                                    title={t("Details")}
+                                    icon="fa-list"
+                                    handler={() => {
```

Comment:
Do we really need the `{` wrapping the function body?
`handler={() => this.props.onSelect(row)}`

And there are others below this

Reply:
Done.